src/framework/preferences/js/Store.js

```diff
@@ -128,13 +131,14 @@ var fluid_3_0_0 = fluid_3_0_0 || {};
         options = options || {};
         var cookieStr = cookieName + "=" + data;
 
-        if (options.expires) {
-            cookieStr += "; expires=" + options.expires;
-        }
-
-        if (options.path) {
-            cookieStr += "; path=" + options.path;
-        }
+        fluid.each(options, function (value, setting) {
+            // skip name because it was already set as the first portion of the cookieStr
+            if (setting !== "name" && setting !== "secure") {
```

Comment:
`HttpOnly` is another no-value cookie setting, similar as `secure`. See [this documentation](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie#Syntax). It's good to handle it too.

Reply:
The cookie store is meant to be used in the browser, as it uses [document.cookie](https://developer.mozilla.org/en-US/docs/Web/API/Document/cookie). If you look at the JavaScript section [Creating Cookies](https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies#Creating_cookies) it says:

> Cookies created via JavaScript cannot include the HttpOnly flag.